NuGet.config

```diff
@@ -1,6 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
+    <add key="dotnet-cli" value="https://www.myget.org/F/dotnet-cli/api/v3/index.json" />
```

Comment:
Do we need this for the `DependencyContext` package? We should mirror these as part of the CoreCLR job so we can gate any breaking behavior.